### PR TITLE
Replace Volley with Glide - login epilogue

### LINKED ISSUE #8088: Replace Volley with Glide - Login Epilogue
Replace Volley with Glide in the Login Epilogue screen. Nothing should change from the user perspective.

To test:
1. Open the app
2. Click on the `Log in` button
3. Enter an email address and click on the `next` button
4. Login either using a magic link or a password
5. Make sure the avatar image in the header is correctly displayed

### PR DESCRIPTION
**Pull request by malinajirka**
Fixes #8088 

Replace Volley with Glide in the Login Epilogue screen. Nothing should change from the user perspective.

To test:
1. Open the app
2. Click on the `Log in` button
3. Enter an email address and click on the `next` button
4. Login either using a magic link or a password
5. Make sure the avatar image in the header is correctly displayed

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginHeaderViewHolder.java [hunk 1]
```diff
@@ -53,6 +55,7 @@ public void updateLoggedInAsHeading(Context context, boolean isAfterLogin, Strin
                 mDisplayNameTextView.setText(username);
             }
         } else {
+            imageManager.cancelRequestAndClearImageView(mAvatarImageView);
```
Comment (kwonye) [inline WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginHeaderViewHolder.java]: Is there a reason we're clearing this? Just trying to think of the use case here on why we'd do this. 

Looks like the only place we're calling this with `isAfterLogin` false is `ShareIntentReceiverFragment.refreshAccountDetails()`

File: WordPress/src/main/res/layout/login_epilogue_header.xml [hunk 2]
```diff
@@ -45,13 +45,13 @@
...
             android:paddingEnd="@dimen/margin_extra_large">
 
-            <org.wordpress.android.widgets.WPNetworkImageView
+            <ImageView
                 android:id="@+id/avatar"
                 android:layout_width="@dimen/avatar_sz_inner_circle"
                 android:layout_height="@dimen/avatar_sz_inner_circle"
                 android:layout_marginTop="@dimen/margin_extra_medium_large"
                 android:layout_marginBottom="@dimen/margin_medium"
-                app:wpDefaultImageDrawable="@drawable/ic_gridicons_user_circle_100dp"/>
```
Comment (kwonye) [inline WordPress/src/main/res/layout/login_epilogue_header.xml]: Is it ok that we're removing the default image here?

Comment (malinajirka) [reply WordPress/src/main/res/layout/login_epilogue_header.xml]: All the placeholder are encapsulated in `ImagePlaceholderManager`, which is used in `ImageManager`.

**Comment by malinajirka** [comment]
Thanks for the review @kwonye! I've answered your comments, let me know if you prefer to remove the `clear...` method invocation.

**Pull request merged**